taxi/views.py

```diff
@@ -52,3 +53,45 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class CarListCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car-list")
+    template = "taxi/car_form.html"
+
+
+class CarDetailUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car-list")
+    template = "taxi/car_form.html"
+
+
+class CarDetailDeleteView(LoginRequiredMixin, generic.DeleteView):
```

Comment:
Why don't you name it just CarCreateView, so on?
You have Detal/List options in view names, you should have consistent naming